colorizer_data/converter.py

```diff
@@ -443,7 +450,9 @@ def convert_colorizer_data(
             be flattened along the Z-axis using a max projection. If `None`, 2D frame generation
             will be skipped.
         frames_3d (Frames3dMetadata | None): A `Frames3dMetadata` object containing the 3D image source
-            ("source") and channel ("segmentation_channel") to use for the 3D image source.
+            ("source") and channel ("segmentation_channel") to use for the 3D image source. The source
+            must be either a URL or a path inside the output dataset directory. If `None`, 3D frames
+            will not be included in the dataset.
```

Comment:
I have an idea for supporting files outside the output dataset directory via symlinks, which I'll open as pt. 3 of this chain.